docs/development/proposing-a-cherry-pick.md

```diff
@@ -0,0 +1,41 @@
+# Cherry-picking changes back to previous branches
+
+## Background
+
+Anyone can propose a cherry-pick - you do not need to be the original author.
+
+We typically have multiple release branches "open" at once - ideally one that is
+a stable release branch, one that is a beta, and one that is an alpha.
+Sometimes the alpha branch will be the `master` branch.
+
+Broad guidelines for acceptance of a cherry-pick:
+
+* Once a version is beta or GA, we typically won't cherry-pick new features back
```

Comment:
Maybe its worth explicitly calling out what _**is**_ typically cherry-picked to these versions? eg: bug fixes, regressions, etc.

Reply:
Agreed with Peter, we've also discussed a few times in office hours what things we make best effort to cherry pick, e.g. new machine-types, new flag support for components like api-server, etc. 

That said, we probably don't want to cover every angle so leaving it loose is probably useful as well 😄 .

Thanks @justinsb